.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -42,7 +48,8 @@ body:
         - Devopness website
         - Devopness web app
         - Documentation
-        - SDK-JS
+        - "@devopness/sdk-js"
+        - "@devopness/ui-react"
```

`@devopness/*` it's a NPM specific notation, correct?
We will in the future support API SDKs for other programming languages, hosted in different package repositories, which might or might not adopt a similar syntax.

Wouldn't be simpler to adopt a human readable version of the module/package name?

```suggestion
        - SDK-JS
        - UI - Design System/React components
```